src/main/java/org/spongepowered/common/service/pagination/ActivePagination.java

```diff
@@ -94,16 +95,19 @@ public UUID getId() {
     }
 
     protected abstract Iterable<Text> getLines(int page) throws CommandException;
+
     protected abstract boolean hasPrevious(int page);
+
     protected abstract boolean hasNext(int page);
+
     protected abstract int getTotalPages();
 
     public void nextPage() throws CommandException {
-        specificPage(++this.currentPage);
+        specificPage(this.currentPage + 1);
```

Comment:
? this doesn't seem to advance currentPage at all -- so keeps on looping the same page.